src/solc_0.8/common/interfaces/IAssetERC1155Token.sol

```diff
@@ -0,0 +1,92 @@
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.2;
+
+interface IAssetERC1155Token {
```

Comment:
This interface doesn't seem to be used anymore

Reply:
@capedcrusader21